bzt/modules/screen.py

```diff
@@ -109,9 +116,20 @@ def _stop(self):
             self.root.destroy()
         super(GUIScreen, self)._stop()
 
+    def change_font(self, event):
+        min_size = 1
+        cur_size = self.font['size']
+        inc = 1 if platform.system() == 'Windows' else -1
```

Comment:
That's not graceful to rely on Windows. Why if one Linux will use positive and another negative? What happens on Mac btw?

Better solution is to look at sign for existing font and use that sign with new value.